client/src/feedbacker-components/comment-panel/comment-panel.js

```diff
@@ -114,7 +115,7 @@ class CommentPanel extends React.Component {
     await this.fetchComments()
     this.scrollToBottom()
 
-    if (!this.props.name) {
+    if (!this.props.name && !this.state.hideName) {
```

Comment:
Why is new state needed, could not hideName from event be used?

Reply:
fixd, this was actually all that was needed